src/deadline/job_attachments/README.md

```diff
@@ -41,6 +41,8 @@ These snapshots are encapsulated in one or more [`asset_manifests`](asset_manife
 
 When starting work, the worker downloads the manifest associated with your job, and recreates the file structure of your submission locally, either downloading all files at once, or as needed if using the [virtual][vfs] job attachments filesystem type. When a task completes, the worker creates a new manifest for any outputs that were specified in the job submission, and uploads the manifest and the outputs back to your S3 bucket.
 
+Manifest files are written locally in the job history directory for debugging purposes (default: `~/.dealine/job_history`).
```

Comment:
There's a bit of impedance mismatch between this comment, that they're "for debugging purposes", and putting them in a directory called `manifests`. I like the idea of placing these manifest files in the job bundles, and think it would be better to make it about that instead of about the job history directory. The job history directory is about storing the history of job bundle submissions. Each job bundle submission includes the manifests that were submitted.

So for this comment, what about changing the description to "Manifest files are written to a `manifests` directory within each job bundle that is added to the job history."? I don't think there's a reason to say it's for debugging purposes, I can think of a lot of other reasons to use these files too.

Another comment to add here is that the file path inside this `manifests` directory corresponds to the S3 manifest path in the submitted job's job attachments metadata.

Reply:
Right good call, I've updated this and mentioned the new flag for CLI that Nathan suggested